README.md

```diff
@@ -252,9 +279,10 @@ JobExecution entity will be have following changed state
 
 ### Finishing raw records parsing
 To indicate end of raw records transfering for parsing should send POST request containing last RawRecordsDto
-to follow path: /change-manager/jobExecutions/{jobExecutionId}/records. 
+to follow path: /change-manager/jobExecutions/{jobExecutionId}/records. \
 {jobExecutionId} - JobExecution id, which can be retrieved from response of JobExecution creation request.
-The last RawRecordsDto should contains empty raw records list ("records" field) and field "last" = true.
+The last RawRecordsDto should contains empty records list ("records" field), appropriate record format value 
```

Comment:
Please describe an aim of the last request - A purpose of the last request is to confirm that there is not more records to upload for the current job. In this case request comes with "last" = true  sign and empty records inside.